backoffice/backoffice/authors/api/views.py

```diff
@@ -140,7 +140,7 @@ def create(self, request):
         airflow_utils.trigger_airflow_dag(
             WORKFLOW_DAGS[workflow.workflow_type].initialize,
             str(workflow.id),
-            workflow.data,
+            workflow=serializer.data,
```

Comment:
I don't understand this chagne, `extra_data` now is `workflow` right? Why we keep the signature the same? https://github.com/inspirehep/inspirehep/pull/3312/files#diff-400fef5e0dec62d9b38fafa4a71749d8dc6d119c66a54301b48d829f903904a8R20

Reply:
not quite, workflow is the full object that comes from django.
`extra_data` is currently only used on the `resolve` endpoint to give intel on how to process it, it will include stuff such as:
```
{
  "value": "accept_curate",
  "create_ticket": false
}
```
we can rename it to something else if it makes it more clear